src/Checkbox/Checkbox.spec.js

```diff
@@ -95,4 +95,9 @@ describe('Checkbox', () => {
     expect(global.console.warn).not.toBeCalled();
     global.console.warn = cachedConsoleWarn;
   });
+
+  it('should not have error class when disabled', () => {
```

Comment:
```suggestion
  it('should not have error when disabled', () => {
```

The fact that it is a class is an implementation detail.